docs/templates/Object Composition/Aggregation/Spatial Decomposition/README.md

```diff
@@ -5,11 +5,11 @@ Provision of a spatial structure of the project by aggregating spatial elements.
 
 > NOTE  The link between the project and the highest level spatial element is provided by this concept through _IfcRelAggregates_ and not through declaration using _IfcRelDeclares_. This is a known anomaly introduced to maintain compatibility with earlier versions of this standard.
 
-The project spatial structure may be made up of a selection of different spatial structure elements with the most generic and simplest form from low to high level as follows: _IfcSpace_, _IfcFacilityPart_ (or _IfcBuildingStorey in the case of buildings), _IfcFacility_ (or any of its subtypes), _IfcSite_ and  _IfcProject_ with _IfcSite_, _FacilityPart_ and _IfcSpace_ being optional levels. Therefore a spatial structure element should only be part of an element at the same or higher level, with the exception of _IfcFacility_ which can be part of an _IfcFacilityPart_ to allow for the regional or longitudinal division of a high level facility in to sections which can then contain one or more smaller functional facilities.
+The project spatial structure may be made up of a selection of different spatial structure elements with the most generic and simplest form from low to high level as follows: _IfcSpace_, _IfcFacilityPart_ (or _IfcBuildingStorey_ in the case of buildings), _IfcFacility_ (or any of its subtypes), _IfcSite_ and  _IfcProject_ with _IfcSite_, _FacilityPart_ and _IfcSpace_ being optional levels. Therefore a spatial structure element should only be part of an element at the same or higher level, with the exception of _IfcFacility_ which can be part of an _IfcFacilityPart_ to allow for the regional or longitudinal division of a high level facility in to sections which can then contain one or more smaller functional facilities.
```

Comment:
_FacilityPart_ should be _IfcFacilityPart_ as well. 

(I am unable to propose changes to the content directly.)

Reply:
That has been captured, but IIRC in a separate PR (to keep changes managed for CEN/ISO review).

Reply:
I hope no merge conflicts will arise from that. 

I withdraw my comment.